.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+node_modules
```

Comment:
- [ OPTIONAL] To ignore node modules, it is better to use `node_modules/`. this means that ignore all files and folders inside the node_modules. using it without the `/` suggest that you are ignoring a file like the `test.md`. For folders, please include the `/`

Reply:
Done